build_scripts/colab-tests/colab-packages.txt

```diff
@@ -0,0 +1,400 @@
+# Generated by running `pip list` inside a frash colab runtime
```

Comment:
do we want an entry on CONTRIBUTE for this process?

Reply:
I would leave it out for now.